src/d3a/models/market/two_sided_pay_as_clear.py

```diff
@@ -27,6 +27,8 @@
 
 log = getLogger(__name__)
 
+Clearing = namedtuple('Clearing', ('rate', 'energy'))
```

Comment:
How about moving this to the market_structures module? That way all market structures will be on the same file.